clang/lib/Serialization/ASTReaderStmt.cpp

```diff
@@ -2135,9 +2135,14 @@ void ASTStmtReader::VisitUnresolvedLookupExpr(UnresolvedLookupExpr *E) {
 
 void ASTStmtReader::VisitTypeTraitExpr(TypeTraitExpr *E) {
   VisitExpr(E);
+  E->TypeTraitExprBits.IsBooleanTypeTrait = Record.readInt();
   E->TypeTraitExprBits.NumArgs = Record.readInt();
   E->TypeTraitExprBits.Kind = Record.readInt();
   E->TypeTraitExprBits.Value = Record.readInt();
```

Comment:
We still want to write/read the `bool` value too?

Reply:
Is there any reason not to?

Reply:
Slight increase of the size of the emitted thing, plus outputing a potentially uninitialized value? (IDK actually if this is set in the other ctor).